platform/nephos/libsaithrift-dev.mk

```diff
@@ -0,0 +1,7 @@
+# libsaithrift-dev package
+
+LIBSAITHRIFT_DEV_NEPHOS = libsaithrift-dev_0.9.4_amd64.deb
```

Comment:
why still 0.9.4?

Reply:
@lguohan , we followed [Broadcom platform](https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/libsaithrift-dev.mk) and saw that each platform vendor using 0.9.4 in this file as well.